docs/docsite/rst/become.rst

```diff
@@ -272,6 +301,9 @@ when UAC is enabled:
       win_reboot:
       when: uac_result|changed
 
+.. Note:: Granting the ``SeTcbPrivilege`` or turning UAC off subverts the
```

Comment:
Please change the last bit of this from “subverts the...” to something like “...can cause Windows security vulnerabilities.”